### PR TITLE
added windows support via cross-env

### PR DESCRIPTION
**Pull request by brian-gates**
Fixes `"NODE_ENV" is not recognized as an internal or external command, operable command or batch file.` on windows

### CODE REVIEW

File: examples/next-prisma-starter-websockets/package.json [hunk 1]
```diff
@@ -5,23 +5,23 @@
...
+    "dev:wss": "cross-env PORT=3001 nodemon --watch src --ext .ts,.tsx,js,jsx --signal SIGTERM --exec \"ts-node --project tsconfig.server.json src/server/wssDevServer.ts\"",
     "dev:next": "next dev",
     "dev:studio": "prisma studio",
     "dev": "yarn migrate-dev && yarn generate && run-p dev:*",
     "dev-nuke": "rm -rf prisma/*.db**",
     "generate": "prisma generate",
     "migrate-dev": "prisma migrate dev",
     "migrate": "prisma migrate deploy",
-    "start": "NODE_ENV=production node dist/server/prodServer.js",
+    1"start": "cross-env NODE_ENV=production node dist/server/prodServer.js",
```
Comment (timcole) [inline examples/next-prisma-starter-websockets/package.json:19]: I believe this `1` is a typo here?

Comment (brian-gates) [reply examples/next-prisma-starter-websockets/package.json]: Thank you!

**Comment by KATT** [comment]
@brian-gates I've added you as a contributor to the project! Could you please push a branch on the trpc origin, close this PR and open a new one? Thanks for doing this!

**Comment by ahhshm** [comment]
Any update from this PR?

**Comment by KATT** [comment]
@ahhshm feel free to take it over.

Also... setup your github sponsors so I can send you some coffee/lunch/beer money for helping out :)

**Comment by ahhshm** [comment]
> I've added you as a contributor to the project! Could you please push a branch on the trpc origin, close this PR and open a new one? Thanks for doing this!

So I close this PR...

> Also... setup your github sponsors so I can send you some coffee/lunch/beer money for helping out :)

@KATT Thank you:) I've discussed why I can't set up github sponsors before. 
> due to the US sanctions on Iran, my country's banks can't connect to the international bank system and I can't have a Paypal account so setting up github sponsors isn't possible for me.

But if you so much want to buy me some coffee, here are my Bitcoin and Ethereum wallets :)
`bitcoin:1KhAK1mBvR7jAKT8QdJHhJorC5o6sXzEsT`
`1KhAK1mBvR7jAKT8QdJHhJorC5o6sXzEsT`

**Pull request closed without merging**